app/src/controllers/plugins/selectors.js

```diff
@@ -36,6 +36,12 @@ export const globalBtsIntegrationsSelector = createSelector(
   (integrations) => filterIntegrationsByGroupType(integrations, BTS_GROUP_TYPE),
 );
 
+export const emailPluginSelector = createSelector(
+  pluginsSelector,
+  (plugins) =>
+    plugins.filter((item) => item.groupType === NOTIFICATION_GROUP_TYPE && item.name === EMAIL)[0],
```

Comment:
Not needed to check `groupType`. Plugin names are unique. Use `find` method here